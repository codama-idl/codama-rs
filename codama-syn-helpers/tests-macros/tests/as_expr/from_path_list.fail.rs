use codama_syn_helpers_test_macros::as_expr;

#[as_expr(foo(1, 2, 3))]
pub struct Test;

#[as_expr(foo = (1, 2, 3))]
pub struct TestWithEq;

fn main() {}
