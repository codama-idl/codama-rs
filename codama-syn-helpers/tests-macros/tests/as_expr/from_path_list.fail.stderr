error: expected a valid expression
 --> tests/as_expr/from_path_list.fail.rs:3:11
  |
3 | #[as_expr(foo(1, 2, 3))]
  |           ^^^^^^^^^^^^

error: expected a valid expression
 --> tests/as_expr/from_path_list.fail.rs:6:11
  |
6 | #[as_expr(foo = (1, 2, 3))]
  |           ^^^^^^^^^^^^^^^
